src/runtime/runtime-constants.ts

```diff
@@ -60,4 +60,6 @@ export const HYDRATED_STYLE_ID = 'sty-id';
 export const HYDRATE_CHILD_ID = 'c-id';
 export const HYDRATED_CSS = '{visibility:hidden}.hydrated{visibility:inherit}';
 
+export const SLOT_FB_CSS = 'slot-fb{display:contents}slot-fb[hidden]{display:none}';
```

Comment:
We have to have two style rules here rather than using `:not([hidden])` since we support some browsers that don't support `:not()` yet

Reply:
Can you do me a favor and add a (code) comment here with that information? That might not be immediately obvious if someone were reading the source file as-is.